src/main/webapp/i18n/de/complaintResponse.json

```diff
@@ -1,5 +1,12 @@
 {
     "artemisApp": {
+        "locks": {
+            "acquired": "Du hast jetzt den Lock auf dieses Complaint für 24 Stunden!",
+            "removeButton": "Entferne Lock",
+            "lockRemoved": "Lock von complaint entfernt",
+            "lockInformation": "Gelocked von {{user}} bis {{endDate}}",
+            "lockInformationYou": "Gelocked von DIR bis {{endDate}}"
+        },
```

Comment:
Vielleicht sollten wir hier eher wie bei dem Assessment selbst "gesperrt" anstatt "gelockt" verwenden.

Reply:
Done in eb25b91fcb559c628c6028cb22dda798874a115d